template.json

```diff
@@ -188,9 +196,6 @@
         "##": "10 = more than 100 min",
         "Time spent for the test (code download to first successful run, [0,10], 10min slots, 100min max)": 0,
 
-        "##": "OS for the test",
-        "Operating system for the test": "",
```

Comment:
Why removing these ?

Reply:
I've moved it to the beginning of the template